package.json

```diff
@@ -45,24 +45,15 @@
     "uglify-js": "^2.4.16",
     "watchify": "^2.2.1"
   },
-  "browser": {
-    "./src/_internals/delegate.js": "./src/_internals/delegateToWorker.js"
-  },
-  "browserify": {
-    "transform": [
-      "./transforms/es7",
-      "./transforms/replace"
-    ]
-  },
   "scripts": {
     "prepublish": "npm run build:node",
     "clean": "rm -rf node/ jsnetworkx.js jsnetworkx-dev.js",
     "build:node": "rm -rf node/ && node scripts/build-node.js src/ node/",
     "build:node:dev": "rm -rf node/ && node scripts/build-node.js --dev src/ node/",
     "watch:node": "npm run build:node:dev -- -w",
-    "build:browser": "(export BUNDLE=jsnetworkx.js; rm -f $BUNDLE && browserify src/browser.js -s jsnx | uglifyjs -cmo $BUNDLE)",
-    "build:browser:dev": "(export BUNDLE=jsnetworkx-dev.js; rm -f $BUNDLE && NODE_ENV=dev browserify src/browser.js -ds jsnx > $BUNDLE)",
-    "watch:browser": "(export BUNDLE=jsnetworkx-dev.js; rm -f $BUNDLE && NODE_ENV=dev watchify src/browser.js -vds jsnx -o $BUNDLE)",
+    "build:browser": "(export BUNDLE=jsnetworkx.js; npm run build:node && rm -f $BUNDLE && browserify node/browser.js -s jsnx | uglifyjs -cmo $BUNDLE)",
+    "build:browser:dev": "(export BUNDLE=jsnetworkx-dev.js; npm run build:node:dev && rm -f $BUNDLE && NODE_ENV=dev browserify node/browser.js -ds jsnx > $BUNDLE)",
+    "watch:browser": "(export BUNDLE=jsnetworkx-dev.js; npm run build:node:dev && rm -f $BUNDLE && NODE_ENV=dev watchify node/browser.js -vds jsnx -o $BUNDLE)",
```

Comment:
I don't think this will work. `watchify` would only be looking at the `node` folder, which does not change when the `src` is edit (unless both, `npm run watch:node` and `npm run watch:browser` are running, which is weird).